### PR TITLE
LG-11409 User Info OIDC Developer Guide Page Rework

### PR DESCRIPTION
**Pull request by nprimak**
Part of the greater epic reworking the developer docs and breaking the side nav up into separate pages

Please refer to the Figma guide to compare design requirements: https://www.figma.com/file/3KSAr0qfDv8BXFIHyFEudZ/Login.gov-OIDC%2C-SAML%2C-and-UX-guides-(LG-10262%2C-LG-10514)?type=design&node-id=104-4001&mode=design&t=YDYdtdA55wzFHwf2-0

### CODE REVIEW

File: _pages/oidc/user-info.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  <div class="desktop:grid-col-8 mobile:grid-col-full">
+    {{ front_matter | markdownify }}
+        <div class="dev-doc-row grid-row">
+            <div class="grid-col-5">
+                <h4 class="parameters">iss <span class="text-normal">(string)</span></h4>
+            </div>
+            <div class="grid-col-7">
+                The issuer of the response, which will be the URL of the Login.gov IdP, for example: <code class="language-plaintext highlighter-rouge">https://idp.int.identitysandbox.gov</code>
+                <ul>
+                    <li>Requires <code class="language-plaintext highlighter-rouge">profile</code> or <code class="language-plaintext highlighter-rouge">profile:name</code> scopes.</li>
```
Comment (karenstrother) [inline _pages/oidc/user-info.md:45]: I recommend removing the period at the end of this sentence for consistency

File: _pages/oidc/user-info.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            </div>
+        </div>
+        <div class="dev-doc-row grid-row">
+            <div class="grid-col-5">
+                <h4 class="parameters">verified_at <span class="text-normal">(number, null)</span></h4>
+            </div>
+            <div class="grid-col-7">
+                When the user's identity was last verified, as an integer timestamp representing the number of seconds since the Unix Epoch, or <code class="language-plaintext highlighter-rouge">null</code> if the account has never been verified.
+                <ul>
+                    <li>Requires the <code class="language-plaintext highlighter-rouge">profile:verified_at</code> scope.</li>
```
Comment (karenstrother) [inline _pages/oidc/user-info.md:78]: Recommend removing this period as well

**Review by anniehirshman-gsa — APPROVED** [review]
Noting the [tiny tab spacing issue](https://github.com/18F/identity-dev-docs/pull/375#issuecomment-1789367947) applies here too - otherwise LGTM! 👍

**Comment by nprimak** [comment]
> Noting the [tiny tab spacing issue](https://github.com/18F/identity-dev-docs/pull/375#issuecomment-1789367947) applies here too - otherwise LGTM! 👍

Yes that should get pulled in when I fix the merge conficts.. I fixed it in the token page before I merged :)

File: _pages/oidc/user-info.md (hunk 2 above)
Comment (nprimak) [reply _pages/oidc/user-info.md]: removed!

File: _pages/oidc/user-info.md (hunk 1 above)
Comment (nprimak) [reply _pages/oidc/user-info.md]: removed!

**Pull request merged**